File: src/problems/palindrome-number/boilerplate-full/function.java
//USER_CODE_HERE
import java.util.*;

public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int x = sc.nextInt();
        Solution sol = new Solution();
        bool result = sol.isPalindrome(x);
        System.out.println(result);
    }
}

File: problems/palindrome-number/boilerplate-full/function.py
#USER_CODE_HERE
import sys
from typing import List

# Set recursion limit higher for recursive problems
sys.setrecursionlimit(2000)

if __name__ == "__main__":
    # Input reading for the specific problem
    x = int(input())

    sol = Solution()
    
    # Call the solution function
    result = sol.isPalindrome(x)
    
    # Print the result
    print(result)